### ISSUE TITLE
lora微调后merge完模型进行lmdeploy推理用时比Qwen2.5-VL-7B-Instruct多一倍，原因为何？

### ISSUE THREAD

**Issue by againaa** [modelscope/ms-swift#4609]
我先进行了lora微调，然后再merge lora，然后用lmdeploy进行https://github.com/modelscope/ms-swift/blob/main/examples/infer/demo.py 方式的推理，但发现merge lora的模型用时比Qwen2.5-VL-7B-Instruct多一倍，请问这个是什么原因呢？
微调代码：
SIZE_FACTOR=8 MAX_PIXELS=65536 CUDA_VISIBLE_DEVICES=0 swift sft \
--model ./Qwen2.5-VL-7B-Instruct \
--train_type lora \
--dataset ./train.json \
--val_dataset ./val.json \
--torch_dtype bfloat16 \
--num_train_epochs 8 \
--per_device_train_batch_size 2 \
--per_device_eval_batch_size 1 \
--learning_rate 1e-4 \
--freeze_vit true \
--lora_rank 8 \
--lora_alpha 32 \
--target_modules all-linear \
--gradient_accumulation_steps 16 \
--eval_steps 50 \
--save_steps 20 \
--save_total_limit 200 \
--logging_steps 5 \
--max_length 8192 \
--output_dir ./output_dir \
--system 'You are a helpful assistant.' \
--warmup_ratio 0.05 \
--dataloader_num_workers 4 \

#merge操作
CUDA_VISIBLE_DEVICES=0 swift infer --ckpt_dir ./output_dir/v17/checkpoint-800 --load_dataset_config true --merge_lora true --val_dataset ./test.json --infer_backend vllm

**Comment by Jintao-Huang** [comment]
推理的时候 也额外加上 SIZE_FACTOR=8 MAX_PIXELS=65536 环境变量

**Comment by againaa** [comment]
> 推理的时候 也额外加上 SIZE_FACTOR=8 MAX_PIXELS=65536 环境变量
加了这个还是不行，时间开销还是没变
SIZE_FACTOR=8 \
MAX_PIXELS=65536 \
CUDA_VISIBLE_DEVICES=0,1,2,3 \
swift infer \
    --model ./output_dir/v15/checkpoint-980-merged \
    --infer_backend lmdeploy \
    --val_dataset './val.json' \
    --dataset_shuffle false \
    --temperature 0.0 \
    --repetition_penalty 1.05 \
    --vision_batch_size 4 \
    --max_new_tokens 4096 \
    --session_len 32768 \
    --result_path '/result.jsonl'

**Comment by againaa** [comment]
> 推理的时候 也额外加上 SIZE_FACTOR=8 MAX_PIXELS=65536 环境变量

是py代码里直接加上这两句吗？但试了下时间是有减少了些，但和用原始的Qwen2.5-VL-7B-Instruct比时间还是多了起码0.5倍
os.environ['SIZE_FACTOR'] = '8'
os.environ['MAX_PIXELS'] = '65536'




完整代码：
import os
import pandas as pd
import csv
import time
os.environ['CUDA_VISIBLE_DEVICES'] = '0,1,2,3,4,5,6,7'
os.environ['SIZE_FACTOR'] = '8'
os.environ['MAX_PIXELS'] = '65536'
def infer_batch(engine: 'InferEngine', infer_requests: List['InferRequest']):
    request_config = RequestConfig(max_tokens=8196, temperature=0,repetition_penalty=1.05,top_p=0.001)
    metric = InferStats()
    resp_list = engine.infer(infer_requests, request_config, metrics=[metric])
    res=[]
    for i in range(len(resp_list)):
        res.append(resp_list[i].choices[0].message.content)
    query0 = infer_requests[0]
    return res
def get_data(img_url):
    txt="请告诉我图中有什么"
    data = {}
    messages = [{'role': 'user', 'content': txt}]
    data['images'] = [img_url]
    data['messages'] = messages
    return data
if __name__ == '__main__':
    from swift.llm import InferEngine, InferRequest, PtEngine, RequestConfig, load_dataset
    from swift.plugin import InferStats
    from swift.llm import LmdeployEngine
    from swift.llm import VllmEngine
    model='./output_dir/v17/checkpoint-800-merge'
    infer_backend = 'lmdeploy'
    df=pd.read_csv('./test.csv')
    dataset_list=[]
    for i in range(len(df['local_path'])):
        img_url=list(df['local_path'])[i]
        data=get_data(img_url)
        dataset_list.append(data)
    batch_size=1024
    t1=time.time()
    for i in range(0, len(dataset_list), batch_size):
        time1=time.time()
        engine = LmdeployEngine(model)#每次都加载，是因为发现第二轮batch开始模型返回结果为空
        dataset = dataset_list[i:i+batch_size]
        infer_requests = [InferRequest(**data) for data in dataset]
        llm_res_list=infer_batch(engine, infer_requests)

**Comment by Jintao-Huang** [comment]
感觉是生成tokens的长度更多了
